batch/batch/publicly_available_images.py

```diff
@@ -2,9 +2,4 @@
 
 
 def publicly_available_images(docker_prefix: str) -> List[str]:
-    # the worker cannot import batch_configuration because it does not have all the environment
-    # variables
-    return [
-        f'{docker_prefix}/{name}'
-        for name in ('hailgenetics/hail', 'hailgenetics/genetics', 'python-dill', 'batch-worker')
-    ]
+    return [f'{docker_prefix}/hailgenetics/{name}' for name in ('hail', 'genetics', 'python-dill')]
```

Comment:
We might need to special case the authentication / docker pull for the batch-worker image to make sure the JVM containers can use it.

Reply:
We already do https://github.com/hail-is/hail/blob/25bafaea4305b21da67f9997bd8b9951a08668b6/batch/batch/worker/worker.py#L432-L433

Reply:
Sorry one last thing. Do we need to add backwards compatibility for `python-dill`?

Reply:
This depends on whether or not you think what was there is a bug. I think it's a bug that we exposed `gcr.io/hail-vdc` to users. Currently, specifying `j.image('hailgenetics/python-dill:3.7')` doesn't work, but specifying `j.image('gcr.io/hail-vdc/python-dill:3.7')` does. It is the only instance where a user can explicitly access an image in `gcr.io/hail-vdc`. It seems really hacky to add in a compatibility layer that specifically rewrites `gcr.io/hail-vdc/python-dill:3.7` to `{DOCKER_PREFIX}/hailgenetics/python-dill:3.7`, especially considering that we're moving off of `gcr.io/hail-vdc` entirely.

Reply:
I feel like an announcement of a breaking bug fix is more appropriate, but if you think this is too much breakage then the other option is to do the rewrite.

Reply:
Let me double check with KC and the rest of our users to see how many people were actually using this.

Reply:
I imagine many were building their own images on top, in which case that is unaffected by this change